### ISSUE TITLE
Train and test set for the pretrained model

### ISSUE THREAD

**Issue by zzhuolun** [gmum/points2nerf#6]
Hi,
very interesting work! 

I'm trying to benchmark and get some metrics (FID, LPIPS, etc) of the generated images from your pretrained model on the shapenet testset.  I'd like to know the train/test split of the pretrained model. Which shapenet objects are in the testset when you train the pretrained model.

Thank you!
